metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email 'cookbooks@chef.io'
 license 'Apache 2.0'
 description 'Manage Splunk Enterprise or Splunk Universal Forwarder'
-version '1.3.1'
+version '1.3.2'
```

Comment:
Please don't modify the version in pull requests. We'll do that when we release a new version.